### PR TITLE
feat: add `/reproduction` slash command

### PR DESCRIPTION
**Pull request by nhedger**
This PR adds a `/reproduction` slash command to the bot to make it easier to ask users for a reproduction.

### CODE REVIEW

**Review by ematipico — APPROVED** [review]

File: src/commands/reproduction.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    Providing a minimal reproduction is important because it helps us quickly understand and diagnose the 
+    issue by isolating the problem to its core components. This makes the debugging process  faster, reduces 
+    misunderstandings, and ensures the issue can be addressed efficiently.
+
+    **What is a good minimal reproduction**
+
+    A good minimal reproduction is a simplified version of your issue that includes only the necessary code 
+    or steps to reproduce the problem. It removes any unrelated features or complexity, focusing solely on 
+    the core issue. This makes it easier for us to quickly identify and fix the problem without unnecessary 
+    distractions.
```
Comment (SuperchupuDev) [inline src/commands/reproduction.ts:28]: can you remove the newlines that are in the middle of sentences? discord will render them

File: src/commands/reproduction.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    `,
+  });
+}
+
+export const slashCommandData = new SlashCommandBuilder()
+  .setName('reproduction')
+  .setDescription('Ask for a reproduction');
```
Comment (SuperchupuDev) [inline src/commands/reproduction.ts:35]:
```suggestion
  .setDescription('Steps to create a minimal reproduction');
```
more descriptive this way i think?

**Review by SuperchupuDev — COMMENTED** [review]
looks good! some small nits:

File: src/commands/reproduction.ts (hunk 2 above)
Comment (nhedger) [reply src/commands/reproduction.ts:29]: Hmm, not sure.

The user running the command is not the actual "target" of the message. The one who runs the command is _asking for a reproduction_, if that makes sense.

**Review by SuperchupuDev — APPROVED** [review]
thanks!

**Comment by nhedger** [comment]
@SuperchupuDev, the new command doesn't seem to appear in prod, even though the bot seems to have been deployed. Did I miss something?

**Comment by SuperchupuDev** [comment]
@nhedger looks like you forgot to import the command in the response handler, hence it shows up but responds with a "not found" message

**Comment by SuperchupuDev** [comment]
i've updated the response handler in 80ea017, should work now

**Pull request merged**